examples/utils/bundle-size-tests/src/odspClient.ts

```diff
@@ -3,7 +3,7 @@
  * Licensed under the MIT License.
  */
 
-import { OdspClient } from "@fluidframework/odsp-client";
+import { OdspClient } from "@fluidframework/odsp-client/internal";
```

Comment:
Is there a rule for when to import from /beta vs. /internal? I would have expected to import from /beta in all cases (i.e. a rule of always import from the most-public option)

Reply:
The rule is that inside FF product /internal is used.
Examples are outside of the product (for the most part) and should not be using /internal. The examples are in general not so healthy and therefore they have just been allowed to use /internal.
The bundle-size-tests are not normal examples and it was easiest to just allow /internal to persist.

The switch to only allow /beta in the `shared-tree-demo` example found an import mistake. One type was being used as another. They happened to have the same type signature; so, build was happy.

Reply:
One thing at a time, but we should ideally be working towards build/linter enforcement that `examples` and `end-to-end-tests` do not use internal APIs. That might be tractable now-ish in `examples` by adding a common base eslint config at the root of the examples directory.